src/components/RichText/RichTextField.tsx

```diff
@@ -234,7 +248,16 @@ export function RichTextField({
                       />
 
                       <FormHelperText id={`${id}-helper-text`}>
```

Comment:
Because `getHelperText` supports a ReactNode, we can do this:

```suggestion
                      <FormHelperText id={`${id}-helper-text`} component="div">
                        {getHelperText(
                          meta,
                          <Box
                            sx={{
                              display: 'flex',
                              justifyContent: 'space-between',
                              mb: 1,
                            }}
                          >
                            <div>{helperText}</div>
                            {showCharacterCount && val ? (
                              <div>
                                <FormattedNumber value={val.characterCount} />{' '}
                                characters
                              </div>
                            ) : undefined}
                          </Box>
```

Reply:
Ok you convinced me lol